src/screens/TopicPageScreen/index.js

```diff
@@ -422,6 +428,7 @@ const TopicPageScreen = (props) => {
         snap
         contentOffset={{x: 0, y: topPosition}}
         contentInsetAdjustmentBehavior={feeds?.length > 1 ? 'automatic' : 'never'}
+        onMomentumScrollEnd={(event) => onWillSendViewPostTime(event, feeds)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `onMomentumScrollEnd` event handler is correct. This change is necessary to send the view post time when the user stops scrolling.
```diff
+ onMomentumScrollEnd={(event) => onWillSendViewPostTime(event, feeds)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed